website/screens/components/grid/code/GridCodePage.tsx

```diff
@@ -211,21 +283,30 @@ const sections = [
           <DxcTable>
             <tr>
               <th>Name</th>
+              <th>Type</th>
+              <th>Description</th>
               <th>Default</th>
-              <HeaderDescriptionCell>Description</HeaderDescriptionCell>
             </tr>
             <tr>
-              <td>areaName: string</td>
-              <td></td>
+              <td>areaName</td>
+              <td>
+                <TableCode>string</TableCode>
+              </td>
               <td>
                 Sets the name of an item so that it can be referenced by a
                 template created with the <Code>grid-template-areas</Code>{" "}
                 property.
               </td>
+              <td>-</td>
             </tr>
             <tr>
-              <td>column: number | string | GridCell</td>
-              <td></td>
+              <td>column</td>
+              <td>
+                <TableCode>
+                  number | string |{" "}
+                  {"{ start: number | string; end: number | string }"}
```

Comment:
In other props as 'placeSelf' the object is broken up in the description, I think we should be consistent with the approach.